### PR TITLE
Avoid running removeDiffSigns on every table

### LINKED ISSUE #765: Issue comment inline code snippets are masked/truncated
With Refined GitHub (incorrect):

screenshot 2017-10-26 11 34 51

---

Without Refined GitHub (correct):

screenshot 2017-10-26 11 34 23

---

Comment for reproduction:

https://github.com/graphile/graphile-build/pull/101#issuecomment-339623628

---

Thanks for this plugin! 🙏

### PR DESCRIPTION
**Pull request by bfred-it**
Fixes #765 

I manually ran the tests described in https://github.com/sindresorhus/refined-github/pull/613

Bug example:

https://github.com/sindresorhus/refined-github/blob/bacb4bcc7aaa7a0052b0b623d1b2fa794b03e339/.gitignore#L1

> Screenshot
> bug

Edit: life would be easier if we had [`ode_modules`](https://www.vocabulary.com/dictionary/ode) instead of `node_modules`

### CODE REVIEW

File: src/content.js [hunk 1]
```diff
@@ -306,15 +306,15 @@ function removeSelectableWhiteSpaceFromDiffs() {
 
 /* Lasciate ogne speranza, voi ch'intrate. */
 function removeDiffSigns() {
-	for (const line of select.all('tr:not(.refined-github-diff-signs)')) {
+	for (const line of select.all('.diff-table tr:not(.refined-github-diff-signs)')) {
```
Comment (bfred-it) [inline src/content.js]: (necessary comment to trigger the bug)

File: src/libs/remove-diff-signs.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+/* Lasciate ogne speranza, voi ch'intrate. */
```
Comment (hkdobrev) [inline src/libs/remove-diff-signs.js]: This is a good time to fix the spelling in this comment.

It should be _ch'**e**ntrate_.

https://en.wikiquote.org/wiki/Inferno_(Dante)#Canto_III:_The_Gate_of_Hell

Comment (bfred-it) [reply src/libs/remove-diff-signs.js]: There's some confusion online, some say `ch'e…` and some say `ch'i…`

This is the most reputable source I found so far and it uses `ch'i…`:

http://etcweb.princeton.edu/cgi-bin/dante/campuscgi/mpb/GetCantoSection.pl?INP_POEM=Inf&INP_SECT=3&INP_START=1&INP_LEN=200&LANG=1&submit=Go

Comment (bfred-it) [reply src/libs/remove-diff-signs.js]: Oh well, I guess this 1350 text from Bocconi University might have to take precedence:



Source: http://amshistorica.unibo.it/170

Comment (bfred-it) [reply src/libs/remove-diff-signs.js]: Also: https://en.wikiquote.org/wiki/Talk:Dante_Alighieri#Ogne_or_Ogni_and_Ch.27intrate_or_Chi.27entrate

**Comment by sindresorhus** [comment]
Great work. Let's hope this is the last of the diff sign churn :)

**Pull request merged**